packages/libs/react-ui/src/components/NavHeader/NavHeaderSelect.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  chevronIconClass,
+  selectClass,
+  selectContainerClass,
+  selectContainerClassDisabled,
+  selectIconClass,
+} from './NavHeader.css';
+
+import { SystemIcon } from '@components/Icon';
+import classNames from 'classnames';
+import type { FC } from 'react';
+import React, { forwardRef } from 'react';
+
+export interface INavHeaderSelectProps
+  extends Omit<
+    React.HTMLAttributes<HTMLSelectElement>,
+    'aria-label' | 'as' | 'className' | 'children' | 'id'
+  > {
+  ariaLabel: string;
+  children: React.ReactNode;
+  disabled?: boolean;
+  icon?: keyof typeof SystemIcon;
+  ref?: React.ForwardedRef<HTMLSelectElement>;
+  onChange?: React.ChangeEventHandler<HTMLSelectElement>;
+  id: string;
+  value?: string;
+}
+
+export const NavHeaderSelect: FC<INavHeaderSelectProps> = forwardRef<
+  HTMLSelectElement,
+  INavHeaderSelectProps
+>(function Select(
+  { ariaLabel, children, disabled = false, onChange, icon, ...rest },
+  ref,
+) {
+  const Icon = icon && SystemIcon[icon];
+  const ChevronDown = SystemIcon.ChevronDown;
+
+  return (
+    <div
+      className={classNames(selectContainerClass, {
+        [selectContainerClassDisabled]: disabled,
+      })}
+      data-testid="kda-select"
```

Comment:
is this being used?

Reply:
If this can be disabled, yes. If we don't need the 'disabled' option at all, then I will remove this. 
@eileenmguo let me know what you think.

Reply:
ahh yes, sorry i did mean the testid